App/Sagas/index.js

```diff
@@ -63,7 +63,7 @@ export default function * root () {
     // takeEvery(StartupTypes.STARTUP, triggerCreateNode),
     // takeEvery(TextileTypes.LOCATION_UPDATE, triggerCreateNode),
     // takeEvery(TextileTypes.BACKGROUND_TASK, triggerCreateNode),
-    takeEvery(TextileTypes.ONBOARDED_SUCCESS, triggerCreateNode),
+    // takeEvery(TextileTypes.ONBOARDED_SUCCESS, triggerCreateNode),
```

Comment:
Don't need this now that we're triggering the app state transition from `TextileManager` (which loads when onboarding is complete)